model/component.js

```diff
@@ -43,24 +43,13 @@ class Component extends ParchKey {
      */
     ports;
 
-
-    /**
-     * The layer on which the Feature Component exists.
-     *
-     * @since 1.0.0
-     * @access public
-     *
-     * @type {string}
-     */
-    layer;
-
     /**
      * The Component Feature to be placed on the Architecture.
      *
      * @since 1.0.0
      * @access public
      *
-     * @type {object}
+     * @type {Array}
```

Comment:
This seems like a regression in the merge, shouldn't this be an object now?

Reply:
You're right it's been changed.